src/pages/Mypage/UserInfo/UserInfo.js

```diff
@@ -1,32 +1,64 @@
 import React from 'react';
-import { SUB_TITLE } from '../SUB_TITLE';
-import BASE_URL from '../../../config';
+import { API } from '../../../config';
 import { Avatar, Card, CardMedia } from '@mui/material';
 import BookmarkIcon from '@mui/icons-material/Bookmark';
 import useFetch from '../../../hooks/useFetch';
 import * as S from './UserInfo.style';
+import { useEffect } from 'react';
+import { useState } from 'react';
 
 const UserInfo = () => {
-  const { loading, data } = useFetch({ BASE_URL });
+  const { loading, data } = useFetch('/data/mypageBid.json');
   const piece = data;
+  const [userInfo, setUserInfo] = useState({});
+
+  useEffect(() => {
+    fetch(`${API.USERS}`)
+      .then(res => res.json())
+      .then(res => setUserInfo(res));
+  }, []);
 
   return (
     <S.CategoryContainer>
       <S.Title>회원 정보</S.Title>
-      <S.UserInfoBox>
-        <S.UserInfoDetail>
-          <Avatar
-            alt="user name"
-            sx={{ width: 70, height: 70, marginRight: 5 }}
-          >
-            정인
-          </Avatar>
-          <S.UserInfoDetailBox>
-            <S.UserName>유정인</S.UserName>
-            <S.UserId>jaylenyu96@gmail.com</S.UserId>
-          </S.UserInfoDetailBox>
-        </S.UserInfoDetail>
-      </S.UserInfoBox>
+      {!(
+        Object.keys(userInfo).length === 0 && userInfo.constructor === Object
+      ) && (
+        <S.UserInfoBox>
+          <S.UserInfoDetail>
+            <Avatar
+              alt="user name"
+              sx={{ width: 70, height: 70, marginRight: 5 }}
+            >
+              {userInfo.user[0].name}
+            </Avatar>
+            <S.UserInfoDetailBox>
+              <S.UserName>{userInfo.user[0].name}</S.UserName>
+              <S.UserId>{userInfo.user[0].email}</S.UserId>
+            </S.UserInfoDetailBox>
+          </S.UserInfoDetail>
+          <S.UserQaulDetail>
+            <S.UserQaulDetailTitle>입찰자격</S.UserQaulDetailTitle>
+            <S.UserQaulDetailText>
+              {userInfo.user[0].bidAgreement}
+            </S.UserQaulDetailText>
+            <S.UserQaulDetailTitle>판매자격</S.UserQaulDetailTitle>
+
+            {userInfo.user[0].bankName === null ? (
+              <S.UserQaulDetailText>등록되지 않았습니다.</S.UserQaulDetailText>
+            ) : (
+              <S.UserQaulDetailText>
+                은행명 : {userInfo.user[0].bankName}
+                <br />
+                계좌번호 : {userInfo.user[0].bankAccount}
+                <br />
+                회원번호 : {userInfo.user[0].artistRegistration}
+              </S.UserQaulDetailText>
+            )}
```

Comment:
```suggestion
const {name,email,bidAgreement,bankName,bankAccount,artistRegisration} = userInfo.user[0]

              {name}
            </Avatar>
            <S.UserInfoDetailBox>
              <S.UserName>{name}</S.UserName>
              <S.UserId>{email}</S.UserId>
            </S.UserInfoDetailBox>
          </S.UserInfoDetail>
          <S.UserQaulDetail>
            <S.UserQaulDetailTitle>입찰자격</S.UserQaulDetailTitle>
            <S.UserQaulDetailText>
              {bidAgreement}
            </S.UserQaulDetailText>
            <S.UserQaulDetailTitle>판매자격</S.UserQaulDetailTitle>

            {bankName === null ? (
              <S.UserQaulDetailText>등록되지 않았습니다.</S.UserQaulDetailText>
            ) : (
              <S.UserQaulDetailText>
                은행명 : {bankName}
                <br />
                계좌번호 : {bankAccount}
                <br />
                회원번호 : {artistRegistration}
              </S.UserQaulDetailText>
            )}
```

반복되는 코드가 있다면 이렇게 줄여보는것도 방법입니다~